test/test_utils.dart

```diff
@@ -0,0 +1,2 @@
+const host = 'localhost';
```

Comment:
Make sure you have the copyright notice everywhere.

```
// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
// for details. All rights reserved. Use of this source code is governed by a
// BSD-style license that can be found in the LICENSE file.
```

Reply:
Done :)